insta/src/components/Login/LoginPage.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Login.css';
+import { Button, Form, FormGroup, Label, Input } from 'reactstrap';
+
+
+class Login extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state={
+            username: '',
+            password: ''
+        };
+    }
+    handleInputChange =e =>{
+     this.setState({ [e.target.name]: e.target.value })
```

Comment:
You can use destructuring here as such. `{name, value} = e.target   this.setState({name: value})`